xdsl/dialects/snitch_runtime.py

```diff
@@ -136,34 +124,46 @@ def __init__(
         )
 
 
+Dma1DWideptrOp = DmaStart1DBaseOp[Annotated[Attribute, u64]]
```

Comment:
Why the Annotated?

Reply:
Because u64 is not a type according to @PapyChacal :smile:

Reply:
Anyways a valid question, because I must admit I didn't fully get what I was writing here, but, well.
It works I guess?